tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
+	[ "${status}" -eq 0 ]
+	echo "${output}" | grep "audio"
```

Comment:
This test is quite loose. I'd prefer that we matched on an expected success message (`group foo added successfully`).

Also, I'd anchor the match:

```
echo "${output}" | egrep "blah blah.*\<audio\>.*blah"
```